### ISSUE TITLE
Re-syndicate when a post is trashed/deleted

### ISSUE THREAD

**Issue by snarfed** [dshanske/syndication-links#184]
Right now, when you move a post to the trash, SL does nothing, but some syndication like Bridgy and Bridgy Fed support deleting syndicated copies. Could SL re-syndicate to those services to trigger those deletions? For those two specifically, it just takes resending the webmention; they detect the deletion when they refetch the post and get a 410 or 404.

Thanks in advance!

**Comment by dshanske** [comment]
This won't work if I switched the plugin to Micropub, will it?

**Comment by snarfed** [comment]
Bridgy Publish supports Micropub delete! https://brid.gy/about#micropub

**Comment by dshanske** [comment]
Your suggestion was specifically re webmentions. I figured.

**Comment by snarfed** [comment]
Not at all! I don't really care about the underlying plumbing, I'd just like to see deletes POSSEd when possible, by any means.

**Comment by dshanske** [comment]
@snarfed When you send the URL, do you send the post URL or the syndicated URL?

**Comment by snarfed** [comment]
Post URL, it's still a standard webmention.

**Comment by snarfed** [comment]
Friendly nudge here! I'd also like to see this for Bridgy Fed. I trashed https://snarfed.org/2024-07-13_dave-slusher-hacker-maker-snarfed-org-i-have-successfu-ellijay-makerspace-social just now, but Synd Links didn't send another wm to BF, so it's still visible under the original Mastodon post, the blank reply here: https://social.ellijaymakerspace.org/@geniodiabolico/112777083077811935 . I'm not at my computer right now so I can't send a wm to BF manually. Would be nice if Synd Links did it automatically!

**Comment by snarfed** [comment]
Related: https://github.com/pfefferle/wordpress-webmention/issues/465
